runtime/ops/bootstrap.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2018-2023 the Deno authors. All rights reserved. MIT license.
+
+use deno_core::op2;
+use deno_core::OpState;
+
+use crate::BootstrapOptions;
+
+deno_core::extension!(
+  deno_bootstrap,
+  ops = [
+    op_bootstrap_args,
+    op_bootstrap_pid,
+    op_bootstrap_numcpus,
+    op_bootstrap_user_agent,
+    op_bootstrap_language,
+    op_bootstrap_log_level,
+    op_bootstrap_no_color,
+  ],
+);
+
+#[op2]
+#[serde]
+pub fn op_bootstrap_args(state: &mut OpState) -> Vec<String> {
+  state.borrow::<BootstrapOptions>().args.clone()
+}
+
+#[op2(fast)]
+#[smi]
+pub fn op_bootstrap_pid() -> u32 {
+  std::process::id()
+}
+
+#[op2(fast)]
+#[smi]
+pub fn op_bootstrap_numcpus(state: &mut OpState) -> u32 {
+  state.borrow::<BootstrapOptions>().cpu_count as u32
+}
+
+#[op2]
+#[string]
+pub fn op_bootstrap_user_agent(state: &mut OpState) -> String {
+  state.borrow::<BootstrapOptions>().user_agent.clone()
+}
```

Comment:
More for information than a request for a change:

Given that we have to clone strings to return them and that the ops are not fast, I think it would be reasonable if you want to return the entire BootstrapOptions as a #[serde] return since it's a one-off at boot time. It might simplify the code overall and I don't _think_ it'll have a measurable impact on boot time.

Reply:
Sure that's an option too. Let's do that afterwards maybe? I don't want to solidify this approach until we get to see some improvements after the removal of bootstrap call? (proabably a few more PRs before that's possible)

Reply:
Sounds good...  merge when you're ready!